### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kohidave**
Update Readme with security note

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,9 @@ The CLI must support workflows around troubleshooting and debugging to help when
 * **Deliver applications continuously.**
 While the ECS CLI can be used to manually deploy changes to an application, we always help customers to move to CI/CD by helping them set up and manage pipelines.
 
+#### Security disclosures
+
+If you think you’ve found a potential security issue, please do not post it in the Issues.  Instead, please follow the instructions [here](https://aws.amazon.com/security/vulnerability-reporting/) or email AWS security directly at [aws-security@amazon.com](mailto:aws-security@amazon.com).
```
Comment (sonofachamp) [inline README.md]: Spacing between sentences is off.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Amazon ECS CLI v2
 
->⚠️ The ECS CLI v2 is currently under active development and is not suitable for production workloads. 
+>⚠️ Before you get started please note that this feature is in preview and the intention is to ensure it meets your requirements and give us any feedback on your use case. Please do not run production workloads till we announce the general availability of this feature. Using the instructions and assets in this repository folder is governed as a preview program under the AWS Service Terms: https://aws.amazon.com/service-terms/ .
```
Comment (sonofachamp) [inline README.md]: Maybe it's just me, but I find the first sentence confusing.

Comment (kohidave) [reply README.md]: It's our "official" line from the containers roadmap. App Sec gave it to me to add.

File: README.md (hunk 1 above)
Comment (kohidave) [reply README.md]: How did you catch that :O

Comment (sonofachamp) [reply README.md]: :man_shrugging: stuck out like a sore :+1:

Comment (kohidave) [reply README.md]: Maaaaaan, is this 30? Not seeing anything anymore?

**Pull request merged**